package.json

```diff
@@ -35,9 +35,11 @@
     "react-messenger-customer-chat": "^0.8.0",
     "react-moment": "^1.1.1",
     "react-perfect-scrollbar": "^1.5.8",
+    "react-smooth-scrollbar": "^8.0.6",
```

Comment:
Replace `react-perfect-scrollbar` to `react-smooth-scrollbar` for better user experience